web/app/js/components/Deployment.jsx

```diff
@@ -89,12 +91,18 @@ export default class Deployment extends React.Component {
         let downstreamMetrics = processRollupMetrics(downstreamRollup.metrics, "targetDeploy");
         let downstreamTsByDeploy = processTimeseriesMetrics(downstreamTimeseries.metrics, "targetDeploy");
 
+        let deploy = _.find(getPodsByDeployment(podList.pods,
+          (componentPods, name) => {
```

Comment:
do we need to have this function as a passed in argument? every place we use getPodsByDeployment we appear to pass in this same function. in that case we should just put it in the main fn.